app/models/user.rb

```diff
@@ -182,6 +184,18 @@ def e164_phone # standard E.164 format used by Twilio
     '+1' + phone.gsub(/\D/, '')
   end
 
+  def office_notification_ids
+    office_users.notifiable.pluck(:id)
+  end
+
+
+  def office_notification_ids=(ids)
+    ids = ids.map(&:to_i)
+    office_users.each do |ou|
+      ou.update!(send_notifications: ou.office_id.in?(ids))
```

Comment:
Code climate says this code doesn't get hit by specs. Run the whole suite locally take a look at simplecov results.

Reply:
Ah yeah, makes sense.